ui/src/schemas/index.ts

```diff
@@ -77,6 +81,26 @@ export const cell = new schema.Entity(
 )
 export const arrayOfCells = [cell]
 
+/* Checks */
+
+// Defines the schema for the "checks" resource
+export const checkSchema = new schema.Entity(
+  ResourceType.Checks,
+  {},
+  {
+    processStrategy: (check: GenCheck): Check => {
+      return {
+        ...check,
+        status: RemoteDataState.Done,
```

Comment:
```suggestion
        status: RemoteDataState.Done,
        tags: check.tags || [], 
```
This was giving us problems before. Is this appropriate ?

Reply:
i think i got a runtime error from this not being there.